src/drivers/trone/trone.cpp

```diff
@@ -586,6 +588,10 @@ TRONE::collect()
 	/* TODO: set proper ID */
 	report.id = 0;
 
+	// This validation check can be used later
```

Comment:
This was already implemented in the previous driver code (before moving to `DISTANCE_SENSOR` msg). It was used  to fill `valid` part of `range_finder` msg. I just reloaded this so it can be used in a later check.